docs/integrations/destinations/bigquery.md

```diff
@@ -190,6 +190,7 @@ The service account does not have the proper permissions.
 - If the GCS staging mode is selected, ensure the BigQuery service account has the right permissions
   to the GCS bucket and path or the `Cloud Storage Admin` role, which includes a superset of the
   required permissions.
+- If you're using per-table permissions, it's important to note that the connector briefly deletes the final tables after a reset. This deletion is sufficient for IAM permissions granted on a single table to be removed along with the table. You can set your permissions at the dataset level or use [authorized datasets/views](https://cloud.google.com/bigquery/docs/authorized-datasets) to access your Airbyte tables.
```

Comment:
```suggestion
- If you're using per-table permissions, it's important to note that the connector deletes the final tables after a reset. This deletion removes IAM permissions granted on a single table to be removed along with the table. You can set your permissions at the dataset level or use [authorized datasets/views](https://cloud.google.com/bigquery/docs/authorized-datasets) to access your Airbyte tables.  [Learn more here](/using-airbyte/core-concepts/typing-deduping)
```

Reply:
It's not the 'temporary' that's the issue - it's that we actually delete the old table and then make a new one ;) 

I've added more docs about this here that we can link to - https://github.com/airbytehq/airbyte/pull/35124